src/warnet/cli/scenarios.py

```diff
@@ -22,18 +22,31 @@ def available():
     """
     console = Console()
     result = rpc_call("scenarios_available", None)
-    if not isinstance(result, list):  # Make mypy happy
-        print(f"Error. Expected list but got {type(result)}: {result}")
-        sys.exit(1)
 
-    # Create the table
-    table = Table(show_header=True, header_style="bold")
-    table.add_column("Name")
-    table.add_column("Description")
+    # main table
+    main_table = Table(show_header=True, header_style="bold magenta")
+    main_table.add_column("Name", style="cyan", no_wrap=True)
+    main_table.add_column("Description", style="green")
```

Comment:
nit: specifying the colours and styles seems like it might cause issues on depending on who is using it? is there a way to just inherit some sane defaults from the user?

Reply:
mmmm, yeah, I could possible make it look more default (worse) 😛

Reply:
These colors come from our CLI lib though, so I think they should handle it ok